src/main/kotlin/snyk/iac/IacIssuesForFile.kt

```diff
@@ -1,12 +1,39 @@
 package snyk.iac
 
-class IacIssuesForFile {
-    lateinit var infrastructureAsCodeIssues: Array<IacIssue>
-    lateinit var targetFile: String
-    lateinit var targetFilePath: String
-    lateinit var packageManager: String
+data class IacIssuesForFile(
+    val infrastructureAsCodeIssues: Array<IacIssue>,
```

Comment:
Just though if we can replace Array with List here to avoid writing equals/hashCode by hands ?

Reply:
Yes, we can even use set, as we don't want duplicates.